openassessment/xblock/static/js/src/studio/oa_edit_fields.js

```diff
@@ -280,4 +280,137 @@ OpenAssessment.DatetimeControl.prototype = {
 
         return errors;
     },
-};
\ No newline at end of file
+};
+
+
+/**
```

Comment:
Nit: I know that ORA is bad about this, but we should use JSDoc conventions for the strings:

https://github.com/edx/edx-platform/wiki/Javascript-standards-for-the-edx-platform